### DISCUSSION TITLE
Why we need start and end params?

### DISCUSSION THREAD

**Discussion by Saloni-59313** [microsoft/presidio#1133] [General]
I was exploring few open source sensitive info masking tool and I came across Presidio.
My doubt is what is the use of this tool when we need to pass the start and end index of the sensitive data?
Is there something which I am missing out or not able to understand?

**Comment by edwardcjohnson** [comment]
you don't have to pass the index of the sensitive sub-string. the start and end index are returned when you use `presidio-analyzer`
